client/src/Components/reviews/Characteristics.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+function Characteristics(props) {
+  return (
+    <div className="characteristicsRating">
+      <h6>Comfort</h6>
+      <div className="ratingSlider">
```

Comment:
please replace this with an array ['Comfort','Fit','Length','Quality'] and a mapping function